sktime/forecasting/fbprophet.py

```diff
@@ -21,8 +21,8 @@ class Prophet(_ProphetAdapter):
     freq: String of DatetimeIndex frequency. See here for possible values:
         https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html
         #timeseries-offset-aliases
-    add_seasonality: Dict with args for Prophet.add_seasonality().
-        Dict can have the following keys/values:
+    add_seasonality: Dict or List of Dicts with args for Prophet.add_seasonality().
+        Dict or each Dict in List can have the following keys/values:
```

Comment:
can we do the same then also for `add_country_holidays`? It is currently also only accepting one country, right? Could you check that @IlyasMoutawwakil , maybe best to change it in equivalence to the changes you did for `add_seasonality`

Reply:
yes, but perhaps better in a diffefent PR? Let's avoid scope creep in PR.

Reply:
Actually this is still an issue in prophet itself. You can't currently add holidays from multiple countries because of overlapping. There's some workaround but it's prophet's [issue](https://github.com/facebook/prophet/issues/909)

Reply:
I guess a consequence of the "overinflated god class" design of prophet... in any case, it means it's out of scope for this PR since it's actually impossible upstream.

I think we really need to add transformers with holiday indicators that work with any estimator, not just prophet, and you can build your own estimator using the pipeline. There is a draft PR on the issue here #1356, though there is surely space for multiple transformers on the topic.